askama_actix/src/lib.rs

```diff
@@ -3,23 +3,23 @@
 pub use askama::*;
 use bytes::BytesMut;
 
-use actix_web::{error::ErrorInternalServerError, HttpResponse};
+use actix_web::{error::ErrorInternalServerError, Error, HttpResponse};
 
 pub trait TemplateToResponse {
-    fn to_response(&self) -> HttpResponse;
+    fn to_response(&self) -> std::result::Result<HttpResponse, Error>;
 }
 
 impl<T: askama::Template> TemplateToResponse for T {
-    fn to_response(&self) -> HttpResponse {
+    fn to_response(&self) -> std::result::Result<HttpResponse, Error> {
```

Comment:
Is `std::result::` needed in here?

Reply:
These are actually needed, because we import everything from `askama` and it exports a `Result` alias which we don't want to use here.